config.yml.default

```diff
@@ -15,6 +15,13 @@ engine:                      # engine settings
     Move Overhead: 100       # increase if your bot flags games too often
     Threads: 2               # max CPU threads the engine can use
     Hash: 256                # max memory (in megabytes) the engine can allocate
+#   go_commands:
+#     searchmoves: "c2c4"    # (Optional) Restrict search to moves in this list.
```

Comment:
I assume that in most positions `c2c4` is illegal.  I'm trying to imagine in what context this configuration setting would be useful.